.circleci/config.yml

```diff
@@ -107,3 +120,4 @@ workflows:
             branches:
               ignore:
                 - main
+      - distribute-dev
```

Comment:
question: Is it intended to distribute a dev version on all branches?

Reply:
No, it's not required for the main branch 🙂